src/main/java/uk/gov/hmcts/reform/idam/web/config/OIDCLocaleChangeInterceptor.java

```diff
@@ -88,15 +89,4 @@ protected void handleException(String localesTag, @NonNull final IllegalArgument
     private LocaleResolver getLocaleResolver(@NotNull final HttpServletRequest request) {
         return RequestContextUtils.getLocaleResolver(request);
     }
-
-    @Nonnull
-    private Set<Locale> getSupportedLocales() {
-        return Arrays.stream(Locale.getISOLanguages())
-            .map(language -> {
-                final URL systemResource = ClassLoader.getSystemResource("messages_" + language + ".properties");
-                return systemResource == null ? null : parseLocaleValue(language);
-            })
-            .filter(Objects::nonNull)
-            .collect(Collectors.toSet());
-    }
```

Comment:
This is the part that doesn't work when in AKS.

Reply:
aha you cant get the resources